### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by bshifaw**
Updated wdl to gatk4.1, Updated imports to git tag URL, updated gatk docker in json, updated Readme structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-#### Setting up parameter json file for a run
-
-To get started, create the json template (using ``java -jar wdltool.jar inputs <workflow>``) for the workflow you wish to run and adjust parameters accordingly.
-
-*Please note that there are optional workflow-level and task-level parameters that do not appear in the template file.  These are set to reasonable values by default, but can also be adjusted if desired.*
-
-#### Required parameters in the somatic panel workflow
+#### Requirements/Expectations
 
 Important: The normal_bams samples in the json can be used test the wdl, they are NOT to be used to create a panel of normals for sequence analysis. For instructions on creating a proper PON please refer to user the documents https://software.broadinstitute.org/gatk/documentation/ .
```
Comment (mwalker174) [inline README.md]: Is there a specific document we can point to? Quick googling turns up a bunch of options, maybe this one is best:
https://gatkforums.broadinstitute.org/gatk/discussion/11053/panel-of-normals-pon

File: README.md [hunk 2]
```diff
@@ -37,7 +29,14 @@ In additional, there are optional workflow-level and task-level parameters that
 
 Further explanation of other task-level parameters may be found by invoking the ``--help`` documentation available in the gatk.jar for each tool.
 
-#### Required parameters in the somatic pair workflow
+#### Outputs 
+- Read count pon in HD5 format
```
Comment (mwalker174) [inline README.md]: pon -> PON

File: README.md [hunk 3]
```diff
@@ -62,8 +61,27 @@ To invoke Oncotator on the called tumor copy-ratio segments:
...
 
-##Important 
-- The data in gs://gatk-test-data/1kgp are from the 1000 Genomes Project (http://www.internationalgenome.org/home) and are provided as is. 
-  If you have questions on the data, please direct them to the 1000 Genomes Project email at info@1000genomes.org. Do NOT post questions about the data to the GATK forum.
+#### Outputs
+- modeled segments for tumor and normal
+- modeled segments plot for tumor and normal
+- denoised copy ratios for tumor and normal
+- denoised copy ratios plot for tumor and normal
+- denoised copy ratios lim 4 plot for tumor and normal
```
Comment (mwalker174) [inline README.md]: Capitalize first words here

File: README.md [hunk 4]
```diff
@@ -1,20 +1,12 @@
 # somatic-cnvs
-Workflows for somatic copy number variant analysis
 
-## Running the Somatic CNV WDL
+### Purpose :
+Workflows for somatic copy number variant analysis.
 
-### Which WDL should you use?
+### cnv_somatic_panel_workflow :
+Builds a panel of normals (PoN) for the cnv pair workflow.
```
Comment (mwalker174) [inline README.md]: Let's use "PON" (capitalized O) to be consistent with this: https://gatkforums.broadinstitute.org/gatk/discussion/11053/panel-of-normals-pon

**Review by mwalker174 — APPROVED** [review]
Thanks for implementing this. I have just a few minor comments for the readme, assuming all the WDL changes come directly from the GATK repo, then feel free to merge. Primarily, I think we should point the user to a specific page to learn more about proper PoN creation. I suggested one but there could be better. @LeeTL1220 are you aware of any detailed documents or discussions on proper PON design?

File: README.md (hunk 1 above)
Comment (bshifaw) [reply README.md]: I can add this one as well, https://gatkforums.broadinstitute.org/dsde/discussion/11682#2

**Pull request merged**